MultimodalQnA/ui/gradio/multimodalqna_ui_gradio.py

```diff
@@ -411,25 +419,25 @@ def select_upload_type(choice, request: gr.Request):
         with gr.Column(scale=3):
             text_upload_result_cap = gr.Textbox(visible=False, interactive=False, label="Upload Status")
         image_upload_cap.upload(ingest_gen_caption, [image_upload_cap, gr.Textbox(value="PDF", visible=False)], [text_upload_result_cap])
-        image_upload_cap.clear(clear_uploaded_file, [], [text_upload_result_cap])
+        image_upload_cap.clear(hide_text, [], [text_upload_result_cap])
 
 with gr.Blocks() as qna:
     state = gr.State(multimodalqna_conv.copy())
     with gr.Row():
         with gr.Column(scale=4):
-            video = gr.Video(height=512, width=512, elem_id="video", visible=True)
-            image = gr.Image(height=512, width=512, elem_id="image", visible=False)
+            video = gr.Video(height=512, width=512, elem_id="video", visible=True, label="Media")
+            image = gr.Image(height=512, width=512, elem_id="image", visible=False, label="Media")
         with gr.Column(scale=7):
             chatbot = gr.Chatbot(elem_id="chatbot", label="MultimodalQnA Chatbot", height=390)
             with gr.Row():
                 with gr.Column(scale=6):
                     # textbox.render()
-                    textbox = gr.MultimodalTextbox(
+                    textbox = gr.Textbox(
                         # show_label=False,
                         # container=False,
                         label="Query",
-                        info="Enter your query here!",
-                        submit_btn=False,
+                        info="Enter a text query here",
```

Comment:
Not sure if this textbox shows up the same way for everyone's browser, but for me I need to click below the text in order to enter the prompt. For a while I was clicking on the "enter your query here" text trying to replace that text with my query, until I realized I need click under the text. If it's like that for others too, it might be more clear to say "Enter a text query below" or something like that. Or if it's possible to have a border on the textbox that could help too. But leaving it how you have it is fine too, just a small issue I wanted to mention.

Reply:
I like that better, thanks